bigframes/session/_io/bigquery/read_gbq_table.py

```diff
@@ -124,6 +125,17 @@ def validate_table(
     if table_ref.dataset_id.startswith("_"):
         return False
 
+    # Materialized views，does not support snapshot
+    if table_type == "MATERIALIZED_VIEW":
+        warnings.warn(
```

Comment:
we should add a test to test this warning

Reply:
Updated.